package.json

```diff
@@ -18,7 +18,8 @@
     "preset": "jest-expo"
   },
   "dependencies": {
-    "@magicbell/react-headless": "5.1.0",
+    "@magicbell/react-headless": "https://pkg.pr.new/@magicbell/react-headless@465",
+    "@magicbell/user-client": "https://pkg.pr.new/@magicbell/user-client@465",
```

Comment:
This PR depends updates to core JS packages:
- https://github.com/magicbell/magicbell-js/pull/465 to allow `token` as a parameter to `MagicBellProvider`
- https://github.com/magicbell/magicbell-js/pull/459 for unwrapping `metadata` and `data` in API responses